src/ensembl/src/content/app/entity-viewer/gene-view/GeneView.tsx

```diff
@@ -344,6 +344,9 @@ const GeneViewWithData = (props: GeneViewWithDataProps) => {
   );
 };
 
+const ensureGeneViewExists = (view: string) =>
```

Comment:
`ensureGeneViewExists` -> `isViewParameterValid`

(the answer to the question in the name of this function would be yes/no, which corresponds to a boolean)

Reply:
✅